main/streams/plain_wrapper.c

```diff
@@ -143,7 +143,7 @@ typedef struct {
 } php_stdio_stream_data;
 #define PHP_STDIOP_GET_FD(anfd, data)	anfd = (data)->file ? fileno((data)->file) : (data)->fd
 
-static int do_fstat(php_stdio_stream_data *d, int force)
+static zend_bool do_fstat(php_stdio_stream_data *d, int force)
```

Comment:
Indeed the underlying fstat is -1,0.
However, this function do_fstat does return only  0,1.
And it's probably best to keep it that way, I think.